Gopkg.lock

```diff
@@ -42,6 +48,11 @@ memo = "3acb5b9124a33a8410d32701c38207f2fcc2c4eec005fd085331a65234904e28"
   version = "v1.1.0"
 
 [[projects]]
+  name = "github.com/docker/docker"
+  packages = ["pkg/term","pkg/term/windows"]
```

Comment:
You could try removing the vendor dir and redoing `dep ensure && dep prune`. I've had that issue before, not sure what it means.